packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx

```diff
@@ -69,13 +65,22 @@ const popover = (onUpdate, errorMessage) => {
     );
 };
 
+const updateSwitches = (switches, setSwitches) => data => {
+    setSwitches({ ...switches, [data.switch_id]: data.is_enabled });
+};
+
+const deleteSwitches = (switches, setSwitches) => switchToRemove => {
+    const { [switchToRemove]: _, ...updatedSwitches } = { ...switches };
```

Comment:
there's no need to create a shallow copy when extracting values:

```javascript
const { [switchToRemove]: _, ...updatedSwitches } = { switches }
```

Also, let's start incorporating comments on more complex code snippets.